### PR TITLE
Do a GET instead of a POST while querying the kubelet.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

File: sources/datasource/kubelet.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func (self *kubeletSource) getContainer(url string, numStats int) (*api.Containe
 	if err != nil {
 		return nil, err
 	}
-	req, err := http.NewRequest("POST", url, bytes.NewBuffer(body))
```
Comment (rjnagal) [inline sources/datasource/kubelet.go]: doh! why did we start with a post?

**Comment by rjnagal** [comment]
LGTM

**Comment by vishh** [comment]
Most likely a refactoring bug.

**Comment by rjnagal** [comment]
travis fails:
   	github.com/GoogleCloudPlatform/heapster/sinks/gcm	[no test files]
--- FAIL: TestWorkflowSuccess (0.00s)
	Location:	cadvisor_test.go:101
	Error:		Not equal: "/b" (expected)
			        != "/a" (actual)
		
	Location:	cadvisor_test.go:102
	Error:		Not equal: "/a" (expected)
			        != "/b" (actual)
		
FAIL
FAIL	github.com/GoogleCloudPlatform/heapster/sources	0.047s

**Comment by vishh** [comment]
The failure is a flake. The test passed this time around. PTAL @rjnagal

**Comment by rjnagal** [comment]
That's a weird flake. Might be a bad test. We should look it up.

**Comment by vishh** [comment]
Done. Sent #139

**Pull request merged**